build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -7,7 +7,6 @@
 steps:
   - label: "build"
     commands:
-      - "./scripts/git/submodule_versions.py init"
```

Comment:
We do still want to initialize submodules here. The thing the script is doing besides messing with SUBMODULE_VERSIONS.txt is getting a shallow copy. The key command is:

`git submodule update --jobs 8 --depth 1`

Maybe you can throw `--init` on there as well. If not then needs to be preceded by `git submodule init`. We could have a shell script that wraps these commands rather than copy-pasting them in every bot, but I don't have strong feelings on that.